webapp/src/app/api/messages/[projectName]/route.ts

```diff
@@ -18,9 +17,8 @@ export async function GET(
     const serverProjectConfig =
       await ServerConfig.getProjectConfig(projectName);
     await RepoGit.cloneIfNotExist(serverProjectConfig);
```

Comment:
I think cloneIfNotExist() after this change can be moved to inside getRepoGit()

Reply:
Yes, but let us do that in a separate merge request.